src/main/scala/beam/utils/scenario/urbansim/censusblock/merger/PlanMerger.scala

```diff
@@ -9,7 +9,7 @@ class PlanMerger(modeMap: Map[String, String]) extends Merger[InputPlanElement,
 
   private def transform(inputPlanElement: InputPlanElement): PlanElement = {
     PlanElement(
-      inputPlanElement.tripId.toString,
+      if (inputPlanElement.tripId.isEmpty) "" else inputPlanElement.tripId.get,
```

Comment:
Could this just be `getOrElse(tripId, "")`?

Reply:
Agree, fixed, thanks!